pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/BuildInfoCollector.java

```diff
@@ -52,15 +53,15 @@
  *         Date: 6/3/17
  */
 @Slf4j
-public class BuildInfoCollector {
+public class BuildInfoCollector implements Closeable {
     private final BuildClient buildClient;
     private final BuildConfigurationClient buildConfigClient;
     private final GroupBuildClient groupBuildClient;
     private final GroupConfigurationClient groupConfigurationClient;
 
-    public void addDependencies(PncBuild bd, String filter) {
+    public static void addDependencies(PncBuild bd, String filter) {
```

Comment:
Why would you make it `static` here? I guess the reason is because it doesn't modify the object state, but what other benefit would it bring?